ci-operator/step-registry/gather/core-dump/gather-core-dump-commands.sh

```diff
@@ -12,5 +12,32 @@ echo "Gathering node core dumps ..."
 mkdir -p ${ARTIFACT_DIR}/core-dumps
 
 oc adm must-gather --dest-dir="${ARTIFACT_DIR}/core-dumps" -- sh -c "/usr/bin/gather_core_dumps || true"
+
+ls -R ${ARTIFACT_DIR}/core-dumps
+CORE_DUMPS="$(find ${ARTIFACT_DIR}/core-dumps -type f)"
+num_core_dumps=$(find ${ARTIFACT_DIR}/core-dumps/*/ -type f | wc -l)
```

Comment:
nit: is there a reason to call `find` again instead of using `$(echo "${CORE_DUMPS}" | wc -l)` in here?

Reply:
just didn't think of it, I guess. more efficient... updated to do it this way.

Reply:
> just didn't think of it, I guess. more efficient... updated to do it this way.

only took me a week 😞 to figure out that the change I added to do it this way `echo -n ${CORE_DUMPS} | grep -c "^"` was exiting with code 1 when there was no lines found. geez. anyway, with a `|| true` now it works as expected.

passing job [here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/20566/rehearse-20566-pull-ci-openshift-ovn-kubernetes-master-4.11-upgrade-from-stable-4.10-e2e-aws-ovn-upgrade/1503943499051962368) with [coredump build-log](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/20566/rehearse-20566-pull-ci-openshift-ovn-kubernetes-master-4.11-upgrade-from-stable-4.10-e2e-aws-ovn-upgrade/1503943499051962368/artifacts/e2e-aws-ovn-upgrade/gather-core-dump/build-log.txt) showing it working. failing job [here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/20566/rehearse-20566-pull-ci-openshift-ovn-kubernetes-release-4.9-4.9-upgrade-from-stable-4.8-e2e-aws-ovn-upgrade/1503847329612959744) because cores are found.

@wking , this ready now.